single-sign-on/index.md

```diff
@@ -25,7 +26,6 @@ The following diagram shows how Single Sign-on (SSO) operates and is managed in
 
 [![SSO architecture](images/sso-architecture.png)](images/sso-architecture.png){: target="_blank" rel="noopener" class="_"}
 
-
 ## Prerequisites
```

Comment:
It looks like prerequisites and everything below it was moved to other topics. Should these sections be removed?

Reply:
Will see if it changes in the new commit. Not seeing the additional sections on my local.

Reply:
Updated